Makefile

```diff
@@ -34,11 +34,11 @@ mockgen: ## generate mock
 	$(GOGEN) ./...
 
 .PHONY: test
-test: ## go test
+test: build ## go test
```

Comment:
[imo]
I think `go test` invokes `go build`, so this dependency run `go build` twice.

Reply:
oops

In https://github.com/dondakeshimo/todo-cli/pull/72, the test passing locally was failed in CI. But after executing `make build`, the test was correctly failed at local. I thought the problem was in Makefile, but it seems incorrect.

Reply:
When I edited one file and just ran test, test result was reflected my editing. So I wrote above opinion. But your situation  doesn't build by some reason (or my situation is too bad not to build...). It must avoid skipping build. So I accept this request.